rust/src/plugin.rs

```diff
@@ -25,3 +27,36 @@ pub fn init() {
         super::debug::SCSetRustLogLevel(super::debug::SCLogGetLogLevel());
     }
 }
+
+// Struct definitions
+#[repr(C)]
+#[allow(non_snake_case)]
+pub struct SCPlugin {
+    pub name: *const libc::c_char,
+    pub license: *const libc::c_char,
+    pub author: *const libc::c_char,
+    pub Init: extern "C" fn(),
+}
```

Comment:
This is a good example of a data structure that should remain in C, and be provided to Rust via bindgen instead of continuing the pattern of creating C data structure in Rust for the convenience of *write once*.